src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,65 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String BUY = "b";
+    private static final String SUPPLY = "s";
+    private static final String WHITE_SPACE = " ";
+
+    protected static int[] readFromFile(String fromFileName) {
```

Comment:
why all methods are `protected static`?

Reply:
because in checklist was recommendation about encapsulation

Reply:
encapsulation is not about static. read once more about access modifiers, for know it's enough to make utility methods private

Reply:
done